rs/crypto/secp256k1/src/lib.rs

```diff
@@ -330,6 +340,48 @@ fn pem_encode(raw: &[u8], label: &'static str) -> String {
     })
 }
 
+/// BIP341 / Taproot derivation step
+///
+/// BIP341 defines a key tweaking operation that occurs with Taproot
+/// <https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#constructing-and-spending-taproot-outputs>
+///
+/// This function implements what is referred to in BIP341 as "tagged_hash"
```

Comment:
Hmm, I don't think that's actually the case. I believe, here, we implement `int_from_bytes(tagged_hash("TapTweak", ...))`, so it's more like `bip341_hash_as_scalar` or `bip341_tweak`.